docs/mp/health/01_introduction.adoc

```diff
@@ -114,24 +141,364 @@ microservice reports itself as ready.
+
+[source,java]
+.Create a new `GreetStartedCheck` class with the following content:
+----
+@Started // <1>
+@ApplicationScoped
+public class GreetStartedCheck implements HealthCheck {
+  private final AtomicLong readyTime = new AtomicLong(0);
+
+
+  @Override
+  public HealthCheckResponse call() {
+    return HealthCheckResponse.named("StartedCheck")  // <2>
+        .status(isStarted())
+        .withData("time", readyTime.get())
+        .build();
+  }
+
+  public void onStartUp(
+      @Observes @Initialized(ApplicationScoped.class) Object init) {
+    readyTime.set(System.currentTimeMillis()); // <3>
+  }
+
+  private boolean isStarted() { // <4>
+    return Duration.ofMillis(System.currentTimeMillis() - readyTime.get()).getSeconds() >= 8;
+  }
+}
+----
+<1> Annotation indicating that this is a startup health check.
+<2> Build the `HealthCheckResponse` with status `UP` after eight seconds, else `DOWN`.
+<3> Record the time at startup of Helidon; the application will declare itself as started eight seconds later.
+<4> Become ready after 5 seconds.
+
+[source,bash]
+.Build and run the application.  Issue the curl command with -v within five seconds and you will see that the application has not yet started:
+----
+curl -v  http://localhost:8080/health/started
+----
+
+[source,json]
+.HTTP response:
+----
+...
+< HTTP/1.1 503 Service Unavailable // <1>
+...
+{
+  "status": "DOWN",
+  "checks": [
+    {
+      "name": "StartedCheck",
+      "status": "DOWN",
+      "data": {
+        "time": 1566399775700
+      }
+    }
+  ]
+}
+----
+<1> The HTTP status is `503` since the application has not started.
+
+[source,bash]
+.After eight seconds you will see the application has started:
+----
+curl -v http://localhost:8080/health/started
+----
+
+[source,json]
+.JSON response:
+----
+...
+< HTTP/1.1 200 OK // <1>
+...
+{
+  "status": "UP",
+  "checks": [
+    {
+      "name": "StartedCheck",
+      "status": "UP",
+      "data": {
+        "time": 1566399775700
+      }
+    }
+  ]
+}
+----
+<1> The HTTP status is `200` indicating that the application is started.
+
+When using the health check URLs, you can get the following health check data:
+
+* liveness only - http://localhost:8080/health/live
+* readiness only -  http://localhost:8080/health/ready
+* startup checks only - http://localhost:8080/health/started
+* all health check data -  http://localhost:8080/health
+
+[source,bash]
+.Get all the health check data, including custom data:
+----
+curl http://localhost:8080/health
+----
+
+[source,json]
+.JSON response:
+----
+{
+  "status": "UP",
+  "checks": [
+    {
+      "name": "LivenessCheck",
+      "status": "UP",
+      "data": {
+        "time": 1566403431536
+      }
+    },
+    ...
+}
+----
 
-You can see all the defaults by accessing any Helidon MP microservice's `/health/live` endpoint
-and viewing the response.
+Full example code is available link:https://github.com/oracle/helidon/tree/master/examples[here].
```

Comment:
1. the link points to examples on master - I think it should point to current release (e.g. a tag in the repository)
2. the link points to examples root, not to a health check example (or at least microprofile examples)

Reply:
1. AFAIK there is no way to give link to tag source tree, just the zipped assets,  so I put the link on branch
2. link to MicroProfile examples.

Reply:
The correct link to a tag is: `https://github.com/oracle/helidon/tree/2.5.1/examples/microprofile`
Your one fix points to a branch which is deleted after the release.
Also please fix the other instances in this document (there are 3 links to examples I think)

Reply:
Fixed.